ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/workers-rhel8/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel8-provision-chain.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/workers-rhel8/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel8-provision-chain.yaml",
```

Comment:
Sorry for the late comment, I suggest to rename `workers-rhel8` to `rhle8` to keep the consistent naming convention.